internal/kafka/internal/presenters/kafka.go

```diff
@@ -114,11 +115,16 @@ func PresentKafkaRequest(kafkaRequest *dbapi.KafkaRequest, kafkaConfig *config.K
 	}, nil
 }
 
-func setBootstrapServerHost(bootstrapServerHost string) string {
-	if bootstrapServerHost != "" {
-		return fmt.Sprintf("%s:443", bootstrapServerHost)
+func setBootstrapServerHost(request *dbapi.KafkaRequest) string {
+	if request.BootstrapServerHost != "" {
+		switch request.Status {
+		case constants.KafkaRequestStatusSuspended.String(), constants.KafkaRequestStatusSuspending.String(), constants.KafkaRequestStatusResuming.String():
```

Comment:
line 121-122 are not unit tested. Can we ensure we've a test for this?

Reply:
thanks @machi1990 ! The lines should be covered now. For both bootstrap and admin server url.